docs/extend-graphql-schema.md

```diff
@@ -3,4 +3,568 @@ id: extend-graphql-schema
+-         countrycodes: "fr",
+-         addressdetails: 1,
+-         q: "foot locker", // for instance…
+-         limit: 10
+-       }
+-     })
+-     .then(response => response.data);
+-
+- const generalOpeningHours = [
+-   "TUESDAY",
+-   "WEDNESDAY",
+-   "THURSDAY",
+-   "FRIDAY",
+-   "SATURDAY"
+- ].map(dayOfWeek => ({
+-   day: dayOfWeek,
+-   openingTimes: [{ from: "9:00", to: "12:30" }, { from: "14:00", to: "20:00" }]
+- }));
+```
+
+```js
+// src/server/modules/physical-stores/loader.js
+import axios from "axios";
+
+const findAllStores = () =>
+  axios
+    .get("https://nominatim.openstreetmap.org/", {
+      params: {
+        format: "json",
+        countrycodes: "fr",
+        addressdetails: 1,
+        q: "foot locker", // for instance…
+        limit: 10
+      }
+    })
+    .then(response => response.data);
+
+const generalOpeningHours = [
+  "TUESDAY",
+  "WEDNESDAY",
+  "THURSDAY",
+  "FRIDAY",
+  "SATURDAY"
+].map(dayOfWeek => ({
+  day: dayOfWeek,
+  openingTimes: [{ from: "9:00", to: "12:30" }, { from: "14:00", to: "20:00" }]
+}));
+
+export default {
+  findAllStores: findAllStores,
+  getGeneralOpeningHours: () => generalOpeningHours
+};
+```
+
+Then, update the module definition to register this new loader as part of your
+module resolver’s context:
+
+```diff
+import typeDefs from "./schema.gql";
+import resolvers from "./resolvers";
++import loader from "./loader";
+
+export default {
+  namespace: "PhysicalStores",
+  typeDefs: typeDefs,
+-  resolvers: resolvers
++  resolvers: resolvers,
++  context: loader
+};
+```
+
+Finally, update the resolvers to use the loader instance injected using
+GraphQL’s context:
+
+```diff
+export default {
+  Query: {
+-    physicalStores: () => findAllStores()
++    physicalStores: (_, __, context) => context.findAllStores()
+  },
+
+  PhysicalStore: {
+    coordinates: store => ({
+      latitude: store.lat,
+      longitude: store.lon
+    }),
+-    openingHours: () => generalOpeningHours
++    openingHours: (_, __, context) => context.getGeneralOpeningHours()
+  },
+```
+
+> **Front-Commerce:** if we were to use the paid version, loaders would have a
+> slightly different ways to be build in order to leverage advanced features
+> useful in production: dependency management / injection, configuration,
+> caching / cache invalidation…
+>
+> _An advanced documentation is coming soon to cover this topic in depth._
+
+## Conclusion
+
+Congratulations! By understanding this simple example you now have all the
+knowledge needed to create your own GraphQL modules.
+
+This is the reason why developers love Front-Commerce: **the GraphQL modular
+middleware makes it very easy to extend and customize Front-Commerce default
+features.** One can write an integration for existing remote services and makes
+it available for frontend developers in a fraction of the time it took them with
+other solutions.
+
+We now recommend you to start experimenting with your existing services! Then,
```

Comment:
I'd make it a bit more actionable and scoped. But I fail to find a nice way to phrase it.
> In order to understand how useful it can be, we recommend you to start experimenting by using your existing services in a custom Front-Commerce module.